src/taskpane/server.js

```diff
@@ -2870,6 +3439,13 @@ function createEntryFromRow(row, model, fieldTypeEnums, artifactIsHierarchical,
             customType = "DecimalValue";
           }
           break;
+        case fieldTypeEnums.customDecimal:

```

Comment:
This case is not handled differently for google and excel. Is that right? Or should a similar pattern to the case for other types be used?

Reply:
Right. However, I kept the functions just in case we need to handle them differently in the future.